src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1266,6 +1265,11 @@
     "create index reports_job_id_idx on reports(job_id) where job_id is not null"
     "create index catalogs_job_id_idx on catalogs(job_id) where job_id is not null"))
 
+(defn add-latest-report-timestamp-to-certnames []
+  (jdbc/do-commands
+    "ALTER TABLE certnames ADD COLUMN latest_report_timestamp timestamp with time zone"
+    "CREATE INDEX idx_certnames_latest_report_timestamp ON certnames(latest_report_timestamp)"))
```

Comment:
The latest_report_id can be NULL... so I think we have to change that or we have to allow this new column to also be NULL and if we allow that then we wouldn't need to backfill.  

We'd need to backfill if we can optimize something else to read this field instead of calculating it in other places.